src/optimistic-oracle/optimistic-oracle.js

```diff
@@ -103,10 +88,10 @@ async function getPrice(contractAddress) {
 
   const price = response.data[tokenAddress][vsCurrency];
 
-  return price;
+  return parseFloat(price);
```

Comment:
this is bad practice for ethereum. you really only want to deal in either `BigNumber` or `String`s since javascript has a max safe value for floats and ints. (99.999999999% of the time safe in this case since no token will be worth that much but still it's very bad practice)

Reply:
Good call, I changed it to use BN all the way until the end, then convert to strings for the finding